### DISCUSSION TITLE
How to get $value in Alert summary

### DISCUSSION THREAD

**Discussion by AreAArseth** [grafana/grafana#45795] [Alerting]
![image](https://user-images.githubusercontent.com/92016247/155350231-18372a7a-d336-42f5-a2b0-caff394d1ea6.png)

I have tried all kind of variants for $value. But none of them return anything. The {{ $labels.dimension }} work fine, but $value/$values etc won't resolve to anything. It's very hard to figure this out by trial and error 👎 How to find the correct spell for extracting the value?

**Comment by AreAArseth** [comment]
Here is the alert pending:
![image](https://user-images.githubusercontent.com/92016247/155350863-5ebb3b43-89f4-40bb-8392-26b56316ecd9.png)
Username {{ $labels.dimension }} is working, but none of the others..

**Comment by pixelrebel** [comment]
I have the same problem.  I'm following the examples in the documentation [here](https://grafana.com/docs/grafana/latest/alerting/fundamentals/annotation-label/how-to-use-labels/)  and [here](https://grafana.com/docs/grafana/latest/alerting/fundamentals/annotation-label/variables-label-annotation/), but the templated string does not render.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
